### ISSUE TITLE
?integrations

### ISSUE THREAD

**Issue by Golab** [twrecked/hass-aarlo#259]
Do you have any plans for making the installation fro integrations in HA Settings?

**Comment by twrecked** [comment]
Yes. I also have plans to try and get flow authorization working for 2FA.

I have no time frame and it's summer here so less coding gets done.
